Vagrantfile.aws_ubuntu-1804

```diff
@@ -0,0 +1,64 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+# workaround to deal with "undefined method `except' for #<Hash:...> (NoMethodError)"
+# see https://github.com/mitchellh/vagrant-aws/issues/566
+class Hash
+  def slice(*keep_keys)
+    h = {}
+    keep_keys.each { |key| h[key] = fetch(key) if has_key?(key) }
+    h
+  end unless Hash.method_defined?(:slice)
+  def except(*less_keys)
+    slice(*keys - less_keys)
+  end unless Hash.method_defined?(:except)
+end
+
+VAGRANTFILE_API_VERSION = "2"
+Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
+
+  # when provisioning symlink /home/ubuntu to /home/vagrant
+  config.vm.provision :shell, path: "vagrant_symlink.sh"
```

Comment:
I think this would work
```
  config.vm.provision :shell, inline: "ln -s /home/ubuntu /home/vagrant"
```
Any reason not to change it to this?
It would allow `vagrant_symlink.sh` to be removed.
According to the docs it runs privileged by default: https://www.vagrantup.com/docs/provisioning/shell#privileged